scio-core/src/main/java/com/google/cloud/dataflow/sdk/io/PatchedAvroIO.java

```diff
@@ -18,6 +18,7 @@
 
```

Comment:
Yes but it'll take a while for them to backport to DataflowJavaSDK so this file will exist for a while.